Window_setup.txt

```diff
@@ -0,0 +1,9 @@
+To run dynamic cli on windows:
+follow the steps!!
+1. first you will require WSL to run dynamic-cli on windows.
+2. https://docs.microsoft.com/en-us/windows/wsl/install
```

Comment:
Can you use hyperlink and add those links there. One more thing link this file in the readme where we are giving instructions for windows user